src/components/results.js

```diff
@@ -8,16 +8,34 @@ export default class Results extends Component {
 		super(props)
 		this.state = {
 			data: {},
-			canVote: 'Yes',
+			canVote: null,
 		};
 	}
 
 	componentDidMount(props) {
 		console.log(this.props)
 		console.log('name: ' + this.props.name);
-		this.getStates()
+		this.yesNoMaybe();
 	}
 
+
+     //Method to determine if you can vote
+
+    yesNoMaybe() {
```

Comment:
Let's change this method name to `userCanVote` or something. @jenessawhite can you think of a better name for this, that follows JS conventions? 

In Ruby, it would probably be `user_can_vote?`. I don't think JavaScript uses questions marks though, right?

Reply:
`userCanVote` works perfect. no question marks, camel case is the standard